rules-reviewed/jboss-eap/orion/xml-orion.windup.xml

```diff
@@ -0,0 +1,68 @@
+        </dependencies>
+        <sourceTechnology id="orion" />
+        <targetTechnology id="eap" versionRange="[6,)" />
+        <tag>reviewed-2015-08-04</tag>
+        <tag>webapp</tag>
+	<tag>orion</tag>
+    </metadata>
+
+    <rules>
+        <rule id="xml-orion-01000">
+            <when>
+                <xmlfile as="default" matches="/orion-ejb-jar">
+                </xmlfile>
+            </when>
+            <perform>
+                <iteration>
+                    <classification title="Oracle Application Platform EJB Descriptor" effort="3" severity="mandatory">
+			<description> Orion specific ejb configuration file used to configure EJBs, map them to jndi names etc.</description>
+		    <link href="https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/6/html/Development_Guide/jboss-ejb3xml_Deployment_Descriptor_Reference.html" title="JBoss-ejb3.xml Deployment Descriptor Reference"/>
+                    <tag>configuration</tag>
+		    </classification>
+                </iteration>
+            </perform>
+        </rule>
+        <rule id="xml-orion-02000">
+            <when>
+                <xmlfile as="default" matches="/orion-web-app">
+                </xmlfile>
+            </when>
+            <perform>
+                <iteration>
+                    <classification title="Oracle Application Platform Web Descriptor" effort="3" severity="mandatory">
+			<description> Oracle Application Platform Web Descriptor configuriation is defined in files by convention named global-web-application.xml and orion-web.xml. It is Orion specific configuration file that besides supporting the standard web.xml features also adds additional support for featuers like buffering, servlet chaining, locales, virtual directories. This file is not supported in JBoss EAP 6 and needs to be migrated to web.xml or JBoss-specific jboss-web.xml</description>
+		    <link href="https://access.redhat.com/documentation/en-US/JBoss_Enterprise_Application_Platform/6.2/html/Security_Guide/jboss-webxml_Configuration_Reference.html" title="JBoss EAP web descriptor"/>
+                    <tag>configuration</tag>
+		    </classification>
+                </iteration>
+            </perform>
+        </rule>
+        <rule id="xml-orion-03000">
+            <when>
+                <xmlfile as="default" matches="/orion-application">
+                </xmlfile>
+            </when>
+            <perform>
+                <iteration>
+                    <classification title="Oracle Application EAR configuration file" effort="3" severity="optional">
+			<description> Orion-application file is an orion specific EAR configuration file used to configure default data sources for CMP beans, security user manager, jndi authorization etc. This file may have been generated by Orion.</description>
+			<link href="TODO" title="JBoss application configuration file"/>
```

Comment:
I suggest looking at the schema: http://www.jboss.org/schema/jbossas/jboss-app_7_0.xsd

It has excellent comments and descriptive elements, IMO.